### PR TITLE
feat(code): add syntax highlighting

### LINKED ISSUE #171: add syntax highlighting component
### Objective
As a user,
When I look at some code in the dashboard,
I see different colors and a copy button
So that I can clearly see what language it is and do the related action fast.

### Acceptance
**Given** I'm looking at the no-release page,
**When** I click on the copy button on the code
**Then** I see that the code has been copied to my clipboard.

**AND**

**Given** I'm looking at the no-release page,
**When** I see some code written to help me,
**Then** I see this code is colorful.

**Notes**
Syntax Highlighting should include the storyscript highlighting.
@wilzbach can you please point here to the storyscript repository that includes the highlighting for the language ?

### PR DESCRIPTION
**Pull request by Arinono**
- [x] Feature
- [x] Unit Tests

Fixes #171

### CODE REVIEW

File: src/components/Code.vue [hunk 1]
```diff
@@ -1,20 +1,95 @@
...
-    <perfect-scrollbar class="max-w-full py-5">
-      <pre><code class="text-white font-monospace"><slot /></code></pre>
+    <perfect-scrollbar class="max-w-full py-5 flex justify-between">
+      <div
+        ref="code"
+        class="font-monospace"
+      >
+        {{ `${prefix ? `${prefix} `: ''}` }}<slot />
+      </div>
+      <div class="flex items-center">
```
Comment (JeanBarriere) [inline src/components/Code.vue]: what happens on the copy if the code is longer than the scrollbar size ?
It stays on right ? With only the `flex items-center` it seems it's not thought of, but I may be mislead.

File: src/main.ts [hunk 2]
```diff
@@ -18,3 +19,5 @@ new Vue({
   }),
   render: h => h(Dashboard)
 }).$mount('#dashboard')
+
+Vue.use(VueClipboard)
```
Comment (JeanBarriere) [inline src/main.ts]: Please use the plugins folder

File: src/components/Code.vue (hunk 1 above)
Comment (Arinono) [reply src/components/Code.vue]: Screenshot 2019-10-08 at 11 28 39

Ace handles himself the line breaking, so there's no more scrolling. Should I find a way to disable the line breaking

Comment (JeanBarriere) [reply src/components/Code.vue]: nope, just remove the flex-center for the copy button

**Pull request merged**